etcdserver/etcdserverpb/rpc.proto

```diff
@@ -911,6 +911,8 @@ message StatusResponse {
   uint64 raftAppliedIndex = 7;
   // errors contains alarm/health information and status.
   repeated string errors = 8;
+  // dbSize is the size of the backend database logically in use, in bytes, of the responding member.
```

Comment:
`s/dbSize/dbSizeInUse/`?

Reply:
i think we should add this as a metrics too. not just putting this into the status response.

Reply:
Fixed.

Also added a metrics.